src/GitVersion.Core/Git/ReferenceName.cs

```diff
@@ -75,9 +80,17 @@ private string RemoveRemote()
     {
         var isRemote = IsPrefixedBy(Canonical, RemoteTrackingBranchPrefix);
 
-        return isRemote
-            ? Friendly.Substring(Friendly.IndexOf("/", StringComparison.Ordinal) + 1)
-            : Friendly;
+        if (isRemote)
+        {
+            var isPullRequest = IsPrefixedBy(Canonical, RemotePullRequestPrefix1)
+                || IsPrefixedBy(Canonical, RemotePullRequestPrefix2);
```

Comment:
Can you perhaps extract `IsPullRequest` to a property so we get a little less repetition among line 28-31 and 85-86?

Reply:
`IsPullRequest` was already a property, so I refactored a little bit the class.